var/spack/repos/builtin/packages/py-ml-collections/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class PyMlCollections(PythonPackage):
+    """ML Collections is a library of Python collections designed for ML usecases."""
+
+    homepage = "https://github.com/google/ml_collections"
+    pypi = "ml_collections/ml_collections-0.1.0.tar.gz"
```

Comment:
I see that the PyPI tarball doesn't include a requirements.txt file so it's broken. I wonder if we should just remove the URL in that case.

Reply:
I'd be inclined to leave it and hopefully they'll fix it in a newer version...